src/ctia/auth/threatgrid.clj

```diff
@@ -68,11 +82,12 @@
                                 :capabilities (->> (str/lower-case role)
                                                    keyword
                                                    (get default-capabilities))}))))
-        auth/denied-identity-singleton)))
+        auth/denied-identity-singleton))))
 
 (defn make-auth-service [get-in-config lookup-stored-identity]
```

Comment:
This function now returns a map instead of an IAuth instance, so it's now a helper for TK.